data/coalition/coalition news.txt

```diff
@@ -341,3 +341,93 @@ news "heliarch expeditions past"
+			`"Just arrived from my first live viewing of the Coalition Games, I have. An incredible experience, it was. Go again next year, I for sure will!"`
+
+news "coalition games planet heliarch"
+	to show
+		month == 7
+	location
+		planet "Far Home"
+	name
+		word
+			"Heliarch agent"
+	message
+		word
+			"Dozens of Heliarch agents are guarding the spaceport entrances, occasionally blocking passage as they try to control the flow of people to and from the Coalition Games events."
+			"You spot a few agents on their lunch break inside a local restaurant. They seem to be enjoying Coalition Games broadcast more than the meals themselves."
+			`"Heading to the Coalition Games, are you? If so, forgive us you must. Some trouble with organizing the crowds this morning, there was, so delayed by five minutes, the whole line is."`
+			`"A former athlete myself, I am. Earned my rank by winning in the Games, I did, so always eager to help organize the event, I am."`
+
+news "coalition games planet people"
+	to show
+		month == 7
+	location
+		planet "Far Home"
+	name
+		word
+			"Saryd captain"
+			"Kimek captain"
+			"Arach captain"
+			"Local Saryd"
+			"Local Kimek"
+			"Local Arach"
+	message
+		word
+			`An avid fan rushes out of a ship next to yours, and into the lines of people being admitted into the Coalition Games. For the next many minutes, an increasingly larger number of Heliarch agents attempt to explain they're not allowed to bring in so many sports goods with them.`
+			`You see a fan angrily stomping to a ship docked nearby, going inside, then coming back out with a coin of sorts. It seems they forgot the entry ticket for the Games, and will have to wait in line all over again.`
+			`"Brought my children to see the Games, I have. An important part of our culture, the Games are, so glad to share the experience with them, I am."`
+			`"Brought my children to see the Games, I have. Hope I do, that behave themselves this time, they will..."`
+			`"Here for the Games too, are you? The largest event in all the Coalition, it is. Come here to watch every year, I do!"`
+			`"Come to watch a relative of mine compete, I have. Training hard for many years, they have been, so sure I am that do well, they will."`
+
+news "coalition games crowd"
+	to show
+		month == 7
+	location
+		planet "Far Home"
+	name
+		word
+			"Coalition Games crowds"
+	message
+		word
+			`The lines of people heading to the Coalition Games seem to have stopped moving abruptly. Upon further investigation, you see some young Kimek have fallen on their backs, being helped back to their feet one by one by Heliarch agents and others from the crowd.`
+			`You arrive to a particularly crowded moment of the spaceport. The lines of people waiting for admission into the Games have now started looping around, heading inside some shops and restaurants, and even a bit into the docking area. Batallions of Heliarch agents work as they can to bring everyone back into orderly lines.`
```

Comment:
```suggestion
			`You arrive to a particularly crowded moment of the spaceport. The lines of people waiting for admission into the Games have now started looping around, heading inside some shops and restaurants, and even a bit into the docking area. Battalions of Heliarch agents work as they can to bring everyone back into orderly lines.`
```
Also, maybe `hard as they can` or some variation of that might sound better there.